src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -463,14 +463,16 @@ class ApplicationManagerImpl
    *
    * @return Current state of the distraction state
    */
-  inline bool driver_distraction() const;
+  inline hmi_apis::Common_DriverDistractionState::eType
+  distracting_driver_state() const;
```

Comment:
@LitvinenkoIra propose to rename to `driver_distraction_state()` / `set_driver_distraction_state()`

Reply:
@AKalinich-Luxoft Done e638bda5ff050f5c1d32aeb2dc575c1016170ca1